apps/server/src/modules/common-cartridge/common-cartridge-client/card-client/card-client.adapter.ts

```diff
@@ -0,0 +1,39 @@
+import { Inject, Injectable, UnauthorizedException } from '@nestjs/common';
+import { extractJwtFromHeader } from '@shared/common';
+import { RawAxiosRequestConfig } from 'axios';
+import { Request } from 'express';
+import { REQUEST } from '@nestjs/core';
+import { BoardCardApi } from './cards-api-client';
+import { CardListResponseDto } from './dto/card-list-response.dto';
+import { CardResponseMapper } from './mapper/card-response.mapper';
+
+@Injectable()
+export class CardClientAdapter {
+	constructor(private readonly boardCardApi: BoardCardApi, @Inject(REQUEST) private request: Request) {}
+
+	public async getAllBoardCardsByIds(cardsIds: Array<string>): Promise<CardListResponseDto> {
```

Comment:
a simple string[] for straightforward cases where you just need a list of strings and preform standard array operations, is a better choice.

Reply:
Done